bbc/simorgh
# Updates to the navigation of Azeri, Serbian, Tigrinya and Turkce

Resolves JIRA n/a

Overall changes
======
- Adds US election topic links Turkce's navigation bar
- Replace Ukraine war on Serbian Latin nav bar with US Election
- Removed topic links on Serbian Cyrillic's nav bar because these topics are stale now that the Serbian team publishes in Optimo
- Add Eritea, Ethiopia, Sudan, MIddle East crisis and US election topics to Tigrinya's nav bar
- Add COP29 to Azeri's nav bar


Code changes
======

- Edited Navigation section of src/app/lib/config/services/azeri.ts to add COPS29 topic link at the end
- Edited Navigation section of src/app/lib/config/services/serbian.ts to add US election topic link in thrid position
 and remove topis from the Cyrillic nav
- Edited Navigation section of src/app/lib/config/services/tigrinya.ts to add five more topic links
- Edited Navigation section of src/app/lib/config/services/turkce.ts to add US election topic link in fourth position
- Updated snapshots

Testing
======
1. Open Azeri front page, last link in the nav should be COP29 and open the COPO29 topic
2. Open Serbian Latin front page, second link in the nav should be US election
4. Open Serbian Cyrillic front page, Home and Most popular should be the only two items in the nav bar
5. Open Tigrinya's front page, Eritrea, Ethiopia, Sudan, Middle east and US election should now be the nav with Video and Most popular at the end,
7. Open Turkce's front page, US Election (ABD Seçimleri) should appear as fourth link in the nav 





![azeri_nav](https://github.com/user-attachments/assets/bd3d0085-22d9-455a-a939-282a6a92a33a)

![serbia_cyr_nav](https://github.com/user-attachments/assets/d67fce87-25f7-40bb-82cf-5aff8d5e7681)
![serbian_lat_nav](https://github.com/user-attachments/assets/09ef15ce-1be0-4dae-aada-3429bdf32185)
![ti_nav](https://github.com/user-attachments/assets/2afbd2d7-76c1-4606-a6c0-e874ee8d14eb)
![tr_nav](https://github.com/user-attachments/assets/dc9a32b2-bad6-4403-be05-a5017cd3a479)



Helpful Links
======
_Add Links to useful resources related to this PR if applicable._

[Coding Standards](https://github.com/bbc/simorgh/blob/latest/docs/Coding-Standards/README.md)

[Repository use guidelines](https://github.com/bbc/simorgh-infrastructure/blob/latest/documentation/repository-guidelines.md)
